docs/copy-backend.md

```diff
@@ -35,6 +37,10 @@ DESCRIPTION
 
     mapping rules
 
+    If a backend with the selected `system-name` is found, it will be
+    updated. Only missing metrics, methods and mapping rules will be created
+    new.
```

Comment:
```suggestion
    .
```

Reply:
applying all changes about doc in a single commit :+1: